app-admin/pinkybar/pinkybar-9999.ebuild

```diff
@@ -0,0 +1,142 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/su8/pinky-bar.git"
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit git-r3 python-single-r1 autotools
+
+DESCRIPTION="Gather some system information and show it in this statusbar program"
+HOMEPAGE="https://github.com/su8/pinky-bar"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+alsa colours cpp drivetemp drivetemp-light dvd libnl lisp lua mpd ncurses
+	+net +pci perl python2 r ruby sensors slang smartemp tcl weather x11"
+
+DEPEND="
+	sys-devel/autoconf
+	>=sys-devel/automake-1.14.1
+	sys-apps/gawk
+	sys-devel/m4
+	dev-lang/perl
+"
+RDEPEND="
+	alsa? ( media-libs/alsa-lib )
+	drivetemp? ( net-misc/curl app-admin/hddtemp )
+	drivetemp-light? ( app-admin/hddtemp )
+	dvd? ( dev-libs/libcdio )
+	libnl? ( >=dev-libs/libnl-3.2.27 virtual/pkgconfig )
+	lisp? ( dev-lisp/ecls )
+	lua? ( dev-lang/lua )
+	mpd? ( media-sound/mpd media-libs/libmpdclient )
+	ncurses? ( sys-libs/ncurses )
+	net? ( sys-apps/iproute2 )
+	pci? ( sys-apps/pciutils )
+	python2? ( ${PYTHON_DEPS} )
+	r? ( dev-lang/R )
+	ruby? ( dev-lang/ruby )
+	sensors? ( sys-apps/lm_sensors )
+	slang? ( sys-libs/slang )
+	smartemp? ( sys-apps/smartmontools )
+	tcl? ( dev-lang/tcl )
+	weather? ( net-misc/curl app-arch/gzip )
+	x11? ( x11-libs/libX11 )
+
+"
+REQUIRED_USE="
+	drivetemp? ( !smartemp )
+	drivetemp-light? ( !smartemp )
+	ncurses? ( !x11 )
+	smartemp? ( !drivetemp !drivetemp-light )
+	x11? ( !ncurses )
+"
+
+src_prepare() {
+	default
+
+	elog 'Generating Makefiles'
+	perl set.pl 'gentoo' || die
+	eautoreconf -if
+}
+
+src_configure() {
+
+	CONFIGURE_OPTS=(
+		$(use_with alsa)
+		$(use_with colours)
+		$(use_with cpp)
+		$(use_with drivetemp)
+		$(use_with drivetemp-light)
+		$(use_with dvd)
+		$(use_with libnl)
+		$(use_with lisp)
+		$(use_with lua)
+		$(use_with mpd)
+		$(use_with ncurses)
+		$(use_with net)
+		$(use_with pci)
+		$(use_with perl)
+		$(use_with python2)
+		$(use_with r)
+		$(use_with ruby)
+		$(use_with sensors)
+		$(use_with slang)
+		$(use_with smartemp)
+		$(use_with tcl)
+		$(use_with weather)
+		$(use_with x11)
+		api_key='28459ae16e4b3a7e5628ff21f4907b6f'
+		icons='/usr/share/icons/xbm_icons'
```

Comment:
Is this a proper fd.o icon set?

Reply:
It's [variable](https://github.com/su8/pinky-bar/blob/master/m4/x11.m4#L42) [used](https://github.com/su8/pinky-bar/blob/master/m4/x11.m4#L67) to point the location where the icons [reside](https://github.com/su8/pinky-bar/blob/master/src/include/non_x11_colours.h#L25). Instead installing them in the $USER home dir, we choose **/usr/share/icons/xbm_icons** and tell that to the program while running `configure`.

Reply:
You didn't answer my question. Do files installed into that directory conform to the fd.o specs? Are they installed into proper subdirs, subject to caching, etc.?

Reply:
I don't know.

Reply:
Looking at the sources, they don't, so they don't belong in `/usr/share/icons`. Also, the LICENSE file for them wrongly assumes they're public-domain — they are in fact all-rights-reserved, unless the original author states otherwise.

Reply:
Okay, can they go in `/usr/share/xbm_icons` then ?

Reply:
Technically, I think it would be best to use `/usr/share/pinkybar/...` to clearly indicate which package is using them. However, given that there's no license allowing redistribution of those icons with pinkybar, they can't be installed.

Reply:
I can drop the **--with-colors** option and it will solve this issue. Anyone that reads the README.md file in the repository can compile the program by hand **--with-colors** option and install the icons on their own. So there is no need to distribute them to the end user this case.